docs/operations/scheduler.md

```diff
@@ -0,0 +1,128 @@
+# KubeVirt Scheduler
+Scheduling is the process of matching Pods/VMs to Nodes. By default, the scheduler used is 
+[kube-scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/).
+Further details can be found at [Kubernetes Scheduler Documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/).
+
+## Creating a Custom Scheduler
+KubeVirt is compatible with custom schedulers. The configuration steps are described in the [Official Kubernetes 
+Documentation](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers).
+Nevertheless, KubeVirt may not be updated at the latest Kubernetes version, and it is required that the custom scheduler
```

Comment:
What do you want to say here exactly? I would put it something like this: The Kubernetes version KubeVirt is running on and the Kubernetes version used to build the custom scheduler have to match.